src/resources/views/inc/form_save_buttons.blade.php

```diff
@@ -1,26 +1,30 @@
 <div id="saveActions" class="form-group">
+    @if(!empty($saveAction))
```

Comment:
Hmm... Can ```$saveAction``` actually be empty? 

It would only be empty if ```create```, ```update``` and ```list``` are all disabled, right? But if they're all disabled, the user _wouldn't have access to the ```create``` or ```update``` forms_. So he can't see the buttons. Am I missing something?

Reply:
Hmm... Maybe I'm being dumb here, but **why would you hide the "save" button for the form, if the user has the ```create```, ```update``` and ```list``` permission?** 

The result, in my opinion, would be a form that _looks_ editable, but _isn't_, because it doesn't have a "save" button. I, as a user, would start editing, only to scroll down and see afterwards that there's no "save" button. Frustrating.

Reply:
@tabacitu ha ha ha basic use case. it's a feature not to required. If anyone want to hide buttons then they can use it. 
Suppose i want to add `Receive Item` button only and other button will be hide. then how can i do that ?? 
Suppose i want to extend `create` view  for my custom url where i didn't want to show all buttons. how can i do that ??
And there are lot of example to need this feature. 
When i work on complex project with backpack i need some flexibility to customize anything depend on use case.